### ISSUE TITLE
Better integration with KDE - e.g. Dolphin as default File Browser (Open, Save as...)

### ISSUE THREAD

**Issue by hrvoj3e** [typora/typora-issues#1751]
Is this possible?

**Comment by abnerlee** [comment]
Sorry, but I cannot understand your request, could you give more details?

**Comment by hrvoj3e** [comment]
I can. Open and "Save as" don't use default OS file browser.
KDE desktop on Linux uses Dolphin as default file browser/explorer. Chrome for examle opens it by default but Typora open another file browser (I think it is GTK related) 

**Typora "File -> Open"**
![screenshot_20180902_114542 typora gtk](https://user-images.githubusercontent.com/4988133/44954704-7d9c4e80-aea6-11e8-985a-5f3799663ccc.png)


**Chrome "Save as..."**
![screenshot_20180902_114753 typora kde dolphin](https://user-images.githubusercontent.com/4988133/44954707-96a4ff80-aea6-11e8-8870-bc87f705f549.png)

**Comment by abnerlee** [comment]
I see, this would be an upstream issue of Electron.

**Comment by litusb** [comment]
Hello everybody,
I don't know for sure if this setback is related with this topic. Here is where I have arrived after googling a while.
I've recently moved to KDE environment and it's the first time I cannot use Typora directly with Google Drive. Dolphin file manager access to Google Drive files via kio-gdrive. I can't reach my files from Typora's file menus because I can't see kio-gdrive link to my Google Drive unit.
